ntbs-build/database-deploy-scripts/2_Deploy_Databases.sh

```diff
@@ -0,0 +1,78 @@
+#!/bin/bash
+
+set -e
+trap 'read -p "Press enter to continue..."' EXIT
+
+ntbs_dir="ntbs_Beta"
+
+migration_dir="ntbs-data-migration"
+migration_sqlproj=".\ntbs-data-migration\ntbs-data-migration.sqlproj"
+migration_dacpac=".\ntbs-data-migration\bin\Debug\ntbs-data-migration.dacpac"
+
+reporting_dir="ntbs-reporting"
+reporting_sqlproj=".\source\ntbs-reporting.sqlproj"
+reporting_dacpac=".\source\bin\Output\ntbs-reporting.dacpac"
+
+specimen_matching_dir="ntbs-specimen-matching"
+specimen_matching_sqlproj=".\source\ntbs-specimen-matching.sqlproj"
+specimen_matching_dacpac=".\source\bin\Debug\ntbs-specimen-matching.dacpac"
+
+read -p "Which environment would you like to deploy to - live, uat, training, int or dev (default uat): " DEPLOY_ENVIRONMENT
```

Comment:
Should we have scripts for the other Azure environments (`test`, `loadtest`, `uat`) as well?

Reply:
There's no reason not to (other than that it was annoying to set them all up). I think the important environments here are `live`, `uat` and `training` because they are the environments that we'll actually deploy releases to. I included `dev` here because that was where I was testing it (and I think I may have run a test against `int` as well). For now, I think I'll remove `int` and `dev` from this list as they might be confusing... but when we move across to using OpenShift/Github I'll make sure to support all of the environments.